pkg/crdexplorer/defaultexplorer/default.go

```diff
@@ -33,12 +35,12 @@ func (e *DefaultExplorer) HookEnabled(kind schema.GroupVersionKind, operationTyp
 			return true
 		}
 	case configv1alpha1.ExploreRetaining:
-		if _, exist := e.retentionHandlers[kind]; exist {
-			return true
-		}
-
-		// TODO(RainbowMango): more cases should be added here
+		// for retaining explorer, always enabled for all GVK.
+		return true
 	}
+	// TODO(RainbowMango): more cases should be added here
+
+	klog.Errorf("Hook is not enabled: %q in %q is not supported.", kind, operationType)
```

Comment:
It's not worth dumping an error here, not all resources need to be customized.

Reply:
how about using debug level logs for debugging?
`klog.V(4).Infof()`

Reply:
+1